stripe.go

```diff
@@ -292,8 +304,15 @@ func (s *BackendImplementation) Do(req *http.Request, body *bytes.Buffer, v inte
 		s.Logger.Printf("Requesting %v %v%v\n", req.Method, req.URL.Host, req.URL.Path)
 	}
 
+	if EnableTelemetry && s.lastRequestMetrics != nil {
+		metricsJSON, _ := json.Marshal(s.lastRequestMetrics)
```

Comment:
This one is unlikely to happen, but let's not swallow errors just in case we introduce something in the future that becomes difficult to debug because we're throwing some message away. Observe `err` and handle if it's non-`nil` like elsewhere.